docs/topics/Gateway.md

```diff
@@ -133,7 +133,7 @@ Once connected, the client should immediately receive an [Opcode 10 Hello](#DOCS
 
 ### Heartbeating
 
-After receiving [Opcode 10 Hello](#DOCS_TOPICS_GATEWAY/hello), the client may begin sending [Opcode 1 Heartbeat](#DOCS_TOPICS_GATEWAY/heartbeat) payloads after `heartbeat_interval * random.random()` milliseconds, and every `heartbeat_interval` milliseconds thereafter. You may send heartbeats before this interval elapses, but you should avoid doing so unless necessary. There is already tolerance in the `heartbeat_interval` that will cover network latency, so you do not need to account for it in your own implementation - waiting the precise interval will suffice.
+After receiving [Opcode 10 Hello](#DOCS_TOPICS_GATEWAY/hello), the client may begin sending [Opcode 1 Heartbeat](#DOCS_TOPICS_GATEWAY/heartbeat) payloads after `heartbeat_interval * jitter` milliseconds (where jitter is between 0 and 1), and every `heartbeat_interval` milliseconds thereafter. You may send heartbeats before this interval elapses, but you should avoid doing so unless necessary. There is already tolerance in the `heartbeat_interval` that will cover network latency, so you do not need to account for it in your own implementation - waiting the precise interval will suffice.
```

Comment:
I feel it's important to mention that jitter is a *random* value between 0 and 1, not just a fixed value that is used every time.

Reply:
Maybe? Maybe not. Depends if you trust the reader to understand that "jitter" implies random

Reply:
Can we just make that into a sentence that explains **you're allowed to start anytime within `heartbeat_interval` miliseconds** from HELLO and then every `heartbeat_interval` miliseconds thereafter.

Reply:
@Mehgugs It should be a random value, because in case of an major outage, you don't want every bot sending heartbeats at the same time when they reconnect.

Reply:
If that is indeed the case, it would probably be a good idea to actually mention that

Reply:
@Mehgugs 

> Can we just make that into a sentence that explains you're allowed to start anytime within heartbeat_interval miliseconds from HELLO and then every heartbeat_interval miliseconds thereafter.

This is more or less what I originally suggested when I reworded this section: https://github.com/discord/discord-api-docs/pull/1527#pullrequestreview-396737905

I'd expected this to be left as an implementation detail to the user; I only wanted to document what the gateway would allow. I.e., the gateway won't close if you *don't* pick a random offset, so I didn't think it was important. But as explained there, the concept of taking a random value was copied from the desktop client behavior.

Aside, I don't personally think its important enough to justify this behavior to the reader. Discord's stability strategies are just an interesting factoid here, and not something that you would take meaningful conclusions / code decisions from.